src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -230,21 +230,31 @@ public void InitializeService(IProtocolEndpoint serviceHost)
                     if ((sqlConn != null) && !connInfo.IsSqlDW && !connInfo.IsAzure)
                     {
                         BackupOperation backupOperation = DisasterRecoveryService.Instance.SetBackupInput(helper.DataContainer, sqlConn, backupParams.BackupInfo);
+                        SqlTask sqlTask = null;
```

Comment:
Why are these static methods that call into a static service? This makes it near-impossible to reliably unit test or integration test, since other runs will impact your own one. You can't parallelize, state errors will break unrelated tests, etc.
- At a minimum, stop having these be static methods and just call SetBackupInput in these methods. I recently did this for the language service